include/truffle.h

```diff
@@ -126,6 +127,8 @@ void truffle_write_idx_b(void *object, int idx, bool value);
 
 // Strings
 void *truffle_read_string(const char *string);
+void *truffle_read_bytes(const char *bytes);
```

Comment:
I think we may just need to document what these functions do. The "string" in truffle_read_string either identifies either the result type (Java String) or the input type (C string), but it reads until a NUL. In truffle_read_bytes we read until a NUL but create a Java byte[]. Both are really conversion style functions, not just "read" style functions.